sections/06-vibrant-bodies.md

```diff
@@ -0,0 +1,59 @@
+
+
+### The Transformative/Material Commons
+
+Stephen Voyce outlines an "open source poetics" based on "a decentralized and nonproprietary model of shared cultural codes, networks of dissemination, and collaborative authorship" (407). His model for open source creativity emphasizes appropriative, readymade, remix, and "uncreative" (referencing Kenneth Goldsmith) strategies, which call into question the role of singular authorship, shifting creative emphasis instead towards contributing to a "poetic commons" (408). In my soundtrack for Izabella Pruska-Oldenhof's _Font Màgica_, at her request, I used a recording of Chopin's Prelude No. 15 ("Raindrop") as a primary source, layering and blending it with recordings of radio astronomy events. These source materials, particularly the Chopin, were algorithmically transformed to draw out the shape and in-between moments of... The homemade Flocking signal processing algorithms... , producing an eerie and slightly melted expressivity
+
+Voyce sees the generative potential in such citational-appropriative techniques, ...
+
+I am interested in the way that software artifacts can be made to support "material linkages" between existing and new works in a manner that goes in some way beyond citation or appropriation. Voyce insightfully imagines that avant-garde poetics can protect and contribute to a "commons"
+
+"The responsibility of the avant-garde will instead require an activistic obligation to create and fortify public domains of open source knowledge, to challenge excessive restrictions placed on language and information, to bring forth marginalized knowledges from a position of inaccessibility to the public at large, and to produce and share artistic tactics and works that challenge intellectual property. That which is at stake is nothing less than open accessibility to culture" (427-8).
+
+
+### The Work as Vector
+
+* Define what open source methods are
+* Ask the question, why do open source, especially when many of the artifacts are personal?--it's unlikely that people will want to fork or remix or reuse this thesis
```

Comment:
For much the same reason that planning for creative networks seems to end up being crucial even when a single human being is involved...

"For much the same reason that lions don't catch fish"